Content.Shared/_Sunrise/PrinterDoc/PrinterDocComponent.cs

```diff
@@ -4,37 +4,101 @@
 using Content.Shared.Containers.ItemSlots;
 using Content.Shared.Materials;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.Serialization;
+using Robust.Shared.Audio;
 
 namespace Content.Shared._Sunrise.PrinterDoc;
 
 [RegisterComponent, NetworkedComponent]
 public sealed partial class PrinterDocComponent : Component
 {
+    [DataField]
+    public Content.Shared._Sunrise.PrinterDoc.PrinterJobView? CurrentJobView;
+
+    [DataField]
+    public SoundSpecifier PrintSound { get; set; } = new SoundPathSpecifier("/Audio/Machines/printer.ogg");
+
+    /// Время выполнения одной печати, копии(в секундах)
+    [DataField]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float JobDuration = 4f;
+
+    [DataField]
+    public int InitialPaperAmount = 700;
+
+    [DataField]
+    public int InitialInkAmount = 7;
+
+    [DataField]
+    public Queue<string> PrintQueue = new();
+
+    /// <summary>
+    /// Очередь заданий на печать или копирование
+    /// </summary>
+    public Queue<(PrinterJobType Type, string? TemplateId)> JobQueue = new();
+
+    /// <summary>
+    /// Идёт ли сейчас выполнение задания
+    /// </summary>
+    [ViewVariables]
+    public bool IsProcessing = false;
+
+    /// <summary>
+    /// ID слота для копируемой бумаги
+    /// </summary>
     public const string CopySlotId = "paper_copy";
 
+    /// <summary>
+    /// Слот для вставляемого листа бумаги, который можно скопировать
+    /// </summary>
     [DataField(required: true)]
     public ItemSlot CopySlot = new();
 
+    /// <summary>
+    /// Прототип материала бумаги (используется для проверки количества бумаги в принтере)
+    /// </summary>
     [DataField(customTypeSerializer: typeof(PrototypeIdSerializer<MaterialPrototype>))]
     [ViewVariables(VVAccess.ReadWrite)]
-    public string PaperMaterial = "Paper";
+    public string PaperMaterial = "officePaper";
```

Comment:
```suggestion
    public ProtoId<MaterialPrototype> PaperMaterial = "officePaper";
```

И убери сверху `customTypeSerializer: typeof(PrototypeIdSerializer<MaterialPrototype>))`